### PR TITLE
Additional files for LM transition model

### PR DESCRIPTION
**Pull request by rois1995**
Hi everyone,

This work updates the transition portion adding config and mesh files for the LM2015 model and the SA-LM coupling.

### CODE REVIEW

**Comment by pcarruscag** [comment]
There are probably some missing files, once you add them change the target branch from master to develop

**Comment by rois1995** [comment]
Can I upload a 3D mesh? Because it is pretty heavy. However, it is freely downloadable from a site. Should I only point it out somewhere?

**Comment by pcarruscag** [comment]
Yes if it is too big link to the source

**Comment by rois1995** [comment]
> Yes if it is too big link to the source

I've put a link in the config file.

**Review by pcarruscag — COMMENTED** [review]
You have config files both here and in the code repo, which ones are you trying to use?
If your meshes are in the tutorials repo you need to use the tutorials.py script to create the regression.

File: compressible_flow/Transitional_Airfoil/Langtry_and_Menter/E387/transitional_SA_LM_model_ConfigFile.cfg [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= RANS
+%
+% Specify turbulent model (NONE, SA, SA_NEG, SST)
+KIND_TURB_MODEL= SA
+KIND_TRANS_MODEL= LM
+FREESTREAM_TURBULENCEINTENSITY= 0.001
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
```
Comment (pcarruscag) [inline compressible_flow/Transitional_Airfoil/Langtry_and_Menter/E387/transitional_SA_LM_model_ConfigFile.cfg]: Clean up the comments here too please.

File: incompressible_flow/Inc_Transitional_Prolate_Spheroid/transitional_SST_LM2015_model_ConfigFile.cfg [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+%
+% Start convergence criteria at iteration number
+CONV_STARTITER= 500
+%
+
+
+% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
+%
+% Mesh input file
+MESH_FILENAME= Mesh.su2
```
Comment (WallyMaier) [inline incompressible_flow/Inc_Transitional_Prolate_Spheroid/transitional_SST_LM2015_model_ConfigFile.cfg:141]: The meshes dont correspond to any in the files

**Pull request merged**